src/components/Calendar/index.js

```diff
@@ -43,9 +44,13 @@ class Calendar extends PureComponent {
     this.estimateMonthSize = this.estimateMonthSize.bind(this);
     this.handleScroll = this.handleScroll.bind(this);
     this.dateOptions = { locale: props.locale };
+    if (this.props.weekStartsOn !== undefined)
+      this.dateOptions.weekStartsOn = this.props.weekStartsOn;
```

Comment:
`this.dateOptions.weekStartsOn = props.weekStartsOn; ` for consistency

Reply:
Added in the last commit